flexget/utils/requests.py

```diff
@@ -238,6 +238,11 @@ def request(self, method, url, *args, **kwargs):
         try:
             log.debug('Fetching URL %s with args %s and kwargs %s', url, args, kwargs)
             result = super(Session, self).request(method, url, *args, **kwargs)
+            try:
+                log.trace('Contents for URL %s start on next line', url)
+                log.trace('%s', result.text)
+            except UnicodeDecodeError:
```

Comment:
There are two problems if this error isn't caught here, and I've added a comment to the code describing them.
1) The call stack unwinds until the error is caught, so turning up the log level causes a change in behaviour, which is bad.
2) This stack trace is dumped to stderr:
Traceback (most recent call last):
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/handlers.py", line 76, in emit
    if self.shouldRollover(record):
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/handlers.py", line 156, in shouldRollover
    msg = "%s\n" % self.format(record)
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 734, in format
    return fmt.format(record)
  File "/Users/johntobin/src/upstream-flexget/flexget/logger.py", line 142, in format
    return logging.Formatter.format(self, record)
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 465, in format
    record.message = record.getMessage()
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 329, in getMessage
    msg = msg % self.args
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 9604: ordinal not in range(128)